docs/flow/modin/experimental/engines/omnisci_on_native/frame/index.rst

```diff
@@ -1,19 +1,18 @@
 :orphan:
 
-OmniSciOnRay Frame Implementation
+OmnisciOnNative Frame Implementation
 =================================
 
-Modin implements ``Frame``, ``PartitionManager``, ``AxisPartition`` and ``Partition`` classes
-specific for ``OmniSciOnRay`` backend:
+Modin implements ``Frame``, ``PartitionManager`` and ``Partition`` classes
+specific for ``OmnisciOnNative`` backend:
 
 * :doc:`Frame <data>`
 * :doc:`Partition <partition>`
-* :doc:`AxisPartition <axis_partition>`
```

Comment:
why `AxisPartition` is gone?

Reply:
`modin/experimental/engines/omnisci_on_ray/frame/axis_partition.py` has been deleted in #3352, so `AxisPartition` documentation is no longer needed.